core/iwasm/aot/aot_runtime.c

```diff
@@ -3059,14 +3049,96 @@ aot_dump_call_stack(WASMExecEnv *exec_env)
         frame.module_offset = 0;
         frame.func_index = cur_frame->func_index;
         frame.func_offset = 0;
+        frame.func_name_wp =
+            get_func_name_from_index(module_inst, cur_frame->func_index);
 
         if (!bh_vector_append(module_inst->frames.ptr, &frame)) {
             bh_vector_destroy(module_inst->frames.ptr);
```

Comment:
Should set module_inst->frame.ptr to NULL, or when next time aot_create_call_stack, L3039, the vector will be destroyed again with invalid pointer.

Reply:
`bh_vector_destroy` doesn't really free the frames.ptr pointer, it just free its internal data and memset the frames.ptr with 0

Reply:
OK, thanks for the notice, had better not change.